### ISSUE TITLE
[BUG] Can't Install Runners in First Launch

### ISSUE THREAD

**Issue by MarceloPenafiel0** [bottlesdevs/Bottles#725]
**Describe the bug**
Bottles freezes when trying to install runners in first launch and fails to install runners

**Installation**
 - Method: Flatpak
 - Version: 2021.11.14-treviso-3

**To Reproduce**
1. Open Bottles (Opening with command line doesn't work either) 
2. Click next  (Welcome window) 
3. Click next (Runners info window)
4. Click next (Almost There window)
5. See error

**Expected behavior**
Downloading window appears and finish runners install.

**Screenshots**
![Screenshot_20211116_142441](https://user-images.githubusercontent.com/58344298/142052400-90f07e14-e06b-4df0-9d7b-055d6a6cb744.png)
I.  Stays there and never install runners

**Desktop:**
 - OS: Kubuntu
 - Version 20.04 LTS
 - Display server: Xorg
 
**Additional context**
Launching through console gives following entries 
```sh
2021-11-16 14:24:16 INFO: Vkd3d found: [vkd3d-v2.4]
2021-11-16 14:24:19 INFO: Bottles Started!
2021-11-16 14:24:31 WARNING: No dxvk found.
2021-11-16 14:24:31 INFO: Vkd3d found: [vkd3d-v2.4]
2021-11-16 14:24:31 WARNING: No nvapi found.
2021-11-16 14:24:31 WARNING: No runners found.
2021-11-16 14:24:31 INFO: Installing component: [vaniglia-6.21].
2021-11-16 14:24:32 INFO: Installing component: [dxvk-nvapi-v0.4].
2021-11-16 14:24:32 INFO: Installing component: [dxvk-1.9.2].
2021-11-16 14:41:59 ERROR: Extraction failed! Archive ends earlier than expected.
2021-11-16 14:41:59 ERROR: Extraction failed! Archive ends earlier than expected.
2021-11-16 14:41:59 WARNING: No nvapi found.
2021-11-16 14:41:59 WARNING: No dxvk found.
2021-11-16 14:41:59 INFO: Installing component: [dxvk-1.9.2].
2021-11-16 14:41:59 INFO: Installing component: [dxvk-nvapi-v0.4].
2021-11-16 14:41:59 ERROR: Extraction failed! Archive ends earlier than expected.
2021-11-16 14:41:59 WARNING: No runners found.
2021-11-16 14:41:59 INFO: Installing component: [vaniglia-6.21].
```

**Comment by pushqrdx** [comment]
Yeah i noticed the same, it's actually downloading something but a) very slow b) no visual feedback, keep restarting it and hopefully it will get through download those 3 components

**Comment by MarceloPenafiel0** [comment]
So should i just leave it there and have faith? Or what can i do?

**Comment by pushqrdx** [comment]
> So should i just leave it there and have faith? Or what can i do?

at least what i did till any of the devs can investigate the issue is that i kept restarting whenever it seemed to take longer than it should be, for instance the `vaniglia` 121 MB, depending on your internet speed you can guesstimate if it's stuck

**Comment by mirkobrombin** [comment]
There is a regression that make installation non-async. I've already fixed in current branch and maybe we should distribute (another) hotfix.

**Comment by mirkobrombin** [comment]
Fixed in 2021.11.14-treviso-4. Will be available soon.
